src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,7 @@
+import { createIconSetFromIcoMoon } from '@expo/vector-icons';
+
+export const Icon = createIconSetFromIcoMoon(
+  require('../../assets/fonts/icomoon/selection.json'),
```

Comment:
Looks like here we can use absolute paths:
Example:
```
require('assets/fonts/icomoon/selection.json')
```

Reply:
You're right